docs/v2-architecture/controller-architecture/guide.md

```diff
@@ -52,14 +81,12 @@ import (
 
 func RegisterTypes(r resource.Registry) {
 	r.Register(resource.Registration{
-		Type:  pbv1alpha1.BarType, 
-		Scope: resource.ScopePartition,
 		Proto: &pbv1alpha1.Bar{},
 	})
 }
 ```
-Note that Scope reference the scope of the new resource, `resource.ScopePartition` 
-mean that resource will be at the partition level and have no namespace, while `resource.ScopeNamespace` mean it will have both a namespace 
+Note that the Scope and Type (Group/Version/Kind) of the resource will be inferred from the protobuf message type using `GetResourceType` and 
+`GetResourceScope` generated methods.mean that resource will be at the partition level and have no namespace, while `pbresource.Scope_SCOPE_NAMESPACE` mean it will have both a namespace 
```

Comment:
```suggestion
`GetResourceScope` generated methods. `pbresource.Scope_SCOPE_PARTITION` means that resource will be at the partition level and have no namespace, while `pbresource.Scope_SCOPE_NAMESPACE` means it will have both a namespace 
```